tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/environment/DockerComposeExecutor.java

```diff
@@ -126,24 +107,26 @@ private void runShellCommand(String[] commands) {
             InputStreamReader isr = new InputStreamReader(stdin);
             BufferedReader br = new BufferedReader(isr);
             String line = null;
-            System.out.println("<OUTPUT>");
+            log.info("<OUTPUT>");
             while ((line = br.readLine()) != null) {
```

Comment:
Could be simplified as `System.out.println(br.lines().collect(Collectors.joining("\n")));`. But there's no need to change.